<launch>
    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find convex_collision_ros)/config/config.rviz" />

    <node pkg="tf2_ros" type="static_transform_publisher" name="static_tf_pub" args="0.75 0 0.25 0 0 0 1 world base" />

    <node name="denavit_hartenberg_demo_node" pkg="convex_collision_ros" type="denavit_hartenberg_demo_node.py" output="screen">
        <rosparam ns="kinematics" subst_value="true" file="$(find convex_collision_ros)/config/awing_frames.yaml" />
        <param name="mesh_path" value="$(find convex_collision_ros)/meshes/awing.obj" />
    </node>
</launch>